tests/i18n/test_percents.py

```diff
@@ -4,19 +4,19 @@
 import os
 
 from django.template import Context, Template
-from django.test import SimpleTestCase, override_settings
+from django.test import override_settings
 from django.utils._os import upath
 from django.utils.encoding import force_text
 from django.utils.translation import activate, get_language, trans_real
 
-from .test_extraction import ExtractorTests
+from .test_extraction import SimplePoFileTests
 
 SAMPLEPROJECT_DIR = os.path.join(os.path.dirname(os.path.abspath(upath(__file__))), 'sampleproject')
 SAMPLEPROJECT_LOCALE = os.path.join(SAMPLEPROJECT_DIR, 'locale')
 
 
 @override_settings(LOCALE_PATHS=[SAMPLEPROJECT_LOCALE])
-class FrenchTestCase(SimpleTestCase):
+class FrenchTestCase(SimplePoFileTests):
```

Comment:
If I'm understanding the original code correctly, `FrenchTestCase` wasn't doing anything before either. Notice it subclassed `django.test.SimpleTestCase`.

AFAICT `FrenchTestCase` was/is simply a placeholder with some extra setup/teardown customization, from which the test class actually doing some concrete test work (`ExtractingStringsWithPercentSigns`) inherits.

Notice also `ExtractingStringsWithPercentSigns` subclassed `ExtractorTests` which is also a base class with no actual test cases, so we aren't reducing the actual number of test cases run with this PR.